.travis/deploy.sh

```diff
@@ -6,3 +6,5 @@ echo "$GPG_OWNERTRUST" | base64 --decode | "$GPG_EXECUTABLE" --import-ownertrust
 set -e
 
 mvn --settings .travis/settings.xml package gpg:sign deploy -Prelease -DskipTests -B -U;
+
+bash .travis/push-javadoc-to-gh-pages.sh
```

Comment:
Maybe it is better to match if publishing needed directly from travis.yml

Reply:
I did it first as a separate deploy provider, but I don't see any cases when publishing javadoc will be untethered from release and drawback of additional deploy step is longer build